config/secrets.yml

```diff
@@ -29,13 +29,12 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-  saml_issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-<%= ENV["SAML_ENV"] %>
-  idp_sso_url: https://idp.<%= ENV["SAML_ENV"] %>.login.gov/api/saml/auth
-  idp_slo_url: https://idp.<%= ENV["SAML_ENV"] %>.login.gov/api/saml/logout
-  http_auth_username: <%= ENV["SP_NAME"] %>
-  http_auth_password: <%= ENV["SP_PASS"] %>
-  # 68:19:... is the fingerprint of the SAML IDP cert CN=int.login.gov
-  # Serial: 0xf41d6ee2e4675981, Issued: July 11 2017, Expires: July 11 2018
-  idp_cert_fingerprint: '68:19:E0:CB:D2:44:A0:CD:B7:D1:80:8A:E2:34:E4:22:90:41:DF:10'
-  acs_url: https://sp.<%= ENV["SAML_ENV"] %>.login.gov/auth/saml/callback
+  <% idp_host = LoginGov::Hostdata.env == 'prod' ? 'secure' : 'idp' %>
+  secret_key_base: <%= ENV.fetch("SECRET_KEY_BASE") %>
+  saml_issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-<%= LoginGov::Hostdata.env %>
+  idp_sso_url: https://<%= idp_host %>.<%= LoginGov::Hostdata.env %>.<%= LoginGov::Hostdata.domain %>/api/saml/auth
```

Comment:
Still not quite right, this renders as `secure.prod.login.gov` rather than `secure.login.gov`.

Reply:
Ah dangit, good catch

Reply:
PTAL @ 24e89ca1f20192fe510ef3a8cf917e85d6ff0c9d